### PR TITLE
change: render paradigm declared in the paradigm

### PR DESCRIPTION
**Pull request by eddieantonio**
# What's in this PR?

The dictionary now uses `Wordform.paradigm` to decided whether it should fetch/render a static paradigm.

- changed: **static paradigms are now explicitly declared**
- changed: `WordformCell`s can now be parsed from layouts
- added: `ParadigmLayout.as_static_paradigm()` returns a paradigm that can be rendered without any further paradigm filling
- added `ParadigmManager.static_paradigm_for()` returns a static paradigm given a name, e.g., from `Wordform.paradigm`
- removed: `ParadigmManager.paradigm_for(analysis)` — we no longer need the analysis to determine which paradigm to fetch

Follow up to #778.

### CODE REVIEW

File: CreeDictionary/CreeDictionary/paradigm/panes.py [hunk 1]
```diff
@@ -139,6 +144,17 @@ def fill(self, forms: dict[str, Collection[str]]) -> Paradigm:
         """
         return Paradigm(pane.fill(forms) for pane in self.panes)
 
+    def as_static_paradigm(self) -> Paradigm:
+        f"""
```
Comment (andrewdotn) [inline CreeDictionary/CreeDictionary/paradigm/panes.py:148]: Is the `f` intentional?
```suggestion
        """
```

**Review by andrewdotn — DISMISSED** [review]

**Review by andrewdotn — COMMENTED** [review]
Looks good, apart from tests failing.

File: CreeDictionary/CreeDictionary/paradigm/panes.py (hunk 1 above)
Comment (eddieantonio) [reply CreeDictionary/CreeDictionary/paradigm/panes.py:148]: Nope 🙃

**Comment by eddieantonio** [comment]
> Looks good, apart from tests failing.

I can't replicate locally! 😭 :sob: 😭 

Looks like this is a Monday problem :/

**Comment by andrewdotn** [comment]
It’ll fail locally if you remove `CreeDictionary/test_db.sqlite3`. Remember that it’s cached on your dev machine, but recreated every time on CI.

Maybe try to set the required `Wordform.paradigm` values in `ensuretestdb`?

Oh! if you’re counting on the migration to set stuff, then it won’t happen on a new import…

**Comment by eddieantonio** [comment]
> It’ll fail locally if you remove `CreeDictionary/test_db.sqlite3`. Remember that it’s cached on your dev machine, but recreated every time on CI.
> 
> Maybe try to set the required `Wordform.paradigm` values in `ensuretestdb`?
> 
> Oh! if you’re counting on the migration to set stuff, then it won’t happen on a new import…

Ahhhh! Thanks for the insight! I will try that! ...on Monday, probably because Friday brain 😅

**Comment by andrewdotn** [comment]
If we re-import the production database, I believe that will also leave us without these new paradigms.

That’s ok because there are already some manual steps for reimports, like redoing auto-translations, and building definition vectors will soon be added to that list.

But if you are feeling ambitious about this particular PR, maybe add a post-import method called by both xmlimport and ensuretestdb?

**Review by andrewdotn — APPROVED** [review]

**Comment by eddieantonio** [comment]
> But if you are feeling ambitious about this particular PR, maybe add a post-import method called by both xmlimport and ensuretestdb?

Exactly what do you mean by "post-import"? Like a simple function call after both, or some kind of signal thingy?

**Comment by andrewdotn** [comment]
Oh, just a simple function call.

**Review by nienna73 — APPROVED** [review]

**Pull request merged**